### PR TITLE
annotate the llama mlir code

### PR DESCRIPTION
**Pull request by effrey-liu**

### CODE REVIEW

**Comment by zhanghb97** [comment]
Thanks! We will iteratively analyze the entire llama model based on this.

File: examples/BuddyLlama/llama_annotation.mlir
Comment (LHY-24) [inline examples/BuddyLlama/llama_annotation.mlir:1]: Good job! I want to make sure if the link of the original Llama model you used here is https://github.com/meta-llama/llama?

Comment (zhanghb97) [reply examples/BuddyLlama/llama_annotation.mlir:1]: @LHY-24 Nice catch! I noticed that the link in our [doc](https://github.com/buddy-compiler/buddy-mlir/tree/main/examples/BuddyLlama#buddy-compiler-llama-example) for llama inference now directs to llama3. We may need to update our llama example README and test the llama3 inference.

Also, @effrey-liu where is your original llama model from?

Comment (effrey-liu) [reply examples/BuddyLlama/llama_annotation.mlir:1]: Model comes from [meta.ai](https://llama.meta.com/), but I chose llama2-7B to generate mlir.

**Pull request merged**